test/integration/targets/template/tasks/main.yml

```diff
@@ -120,6 +120,74 @@
         - 'import_as_with_context_diff_result.stdout == ""'
         - "import_as_with_context_diff_result.rc == 0"
 
+# VERIFY trim_blocks
+
+- name: Render a template with "trim_blocks" set to False
+  template: src=trim_blocks.j2 dest={{output_dir}}/trim_blocks_false.templated trim_blocks=False
+
+- name: copy known good trim_blocks_false.expected into place
+  copy: src=trim_blocks_false.expected dest={{output_dir}}/trim_blocks_false.expected
+
+- name: compare templated file to known good trim_blocks_false
+  shell: diff -u {{output_dir}}/trim_blocks_false.templated {{output_dir}}/trim_blocks_false.expected
```

Comment:
Rather than a `shell` task, you can use the `stat` module and `assert` that the `checksum` matches. See [lineinfile tests](https://github.com/ansible/ansible/blob/devel/test/integration/targets/lineinfile/tasks/main.yml#L58) for examples.